tests/non-js-resources/subtests/service-worker/parcel/index.md

```diff
@@ -1,8 +1,5 @@
 ---
-result: fail
-issue:
-  - https://github.com/parcel-bundler/parcel/issues/4227
-  - https://github.com/parcel-bundler/parcel/issues/3661
+result: pass
 ---
 
-Parcel handles ServiceWorkers correctly, but doesn’t seem to give you access to the list of assets.
+Parcel handles service workers out of the box. `@parcel/service-worker` can be used to access a list of hashed asset URLs.
```

Comment:
```suggestion
Parcel handles ServiceWorkers out of the box. `@parcel/service-worker` can be used to access a list of hashed asset URLs.
```

I’m not sure if we have ever canonicalized one spelling or the other on here... @jakearchibald ?

Reply:
Also `@parcel/service-worker` is nice!!

Reply:
looks like most of the other references in the repo have spaces, but are capitalized